tests/lib/policy_test.py

```diff
@@ -437,6 +437,11 @@
   source-zone:: zone1 zone2
   destination-zone:: zone1 zone2
   action:: accept
+"""
```

Comment:
It looks like the test failures are actually due to a close bracket getting effectively deleted here. Please fix this and rebase for us to merge.

Reply:
Damn, didn't spot this, fixed :-)